labelbox/client.py

```diff
@@ -636,16 +637,33 @@ def create_project(self, **kwargs) -> Project:
             description (str): A short summary for the project
             media_type (MediaType): The type of assets that this project will accept
             queue_mode (Optional[QueueMode]): The queue mode to use
-            auto_audit_percentage (Optional[float]): The percentage of data rows that will require more than 1 label
-            auto_audit_number_of_labels (Optional[float]): Number of labels required for data rows selected for multiple labeling (auto_audit_percentage)
+            quality_mode (Optional[QualityMode]): The quality mode to use (e.g. Benchmark, Consensus). Defaults to
```

Comment:
what happens to existing projects where these attributes are set and, say, we do smth like `client.get_project()`. I suppose it would work? But from the client point of view, is there any migration needs to happen, i.e. if auto_audit_percentage and auto_audit_number_of_labels => set quality mode etcv

Reply:
Also if the quality mode default to Benchmark, shouldn't we set it here?

Reply:
`auto_audit_percentage` and `auto_audit_number_of_labels` are still in the Project model but have no meaning anymore - it has been replaced with equivalents at the batch level (specified during batch creation). The migration has already been performed on the backend, this change simply removes the old obsolete data from the client model